ociinstaller/tempdir.go

```diff
@@ -21,7 +23,7 @@ func NewTempDir(path string) *tempDir {
 }
 
 func getOrCreateTempDir(ref string) string {
-	pluginCacheDir := filepath.Join(constants.TempDir(), safeDirName(ref))
+	pluginCacheDir := filepath.Join(constants.PluginDir(), safeDirName(fmt.Sprintf("tmp-%s", generateTempDirName())))
```

Comment:
do we delete this dire afterwards?

Reply:
Yes.

In `ociinstaller/plugin.go` `#L18`